  Debbugger fails to show variables in 2016.2 (PY-20198)  Fallback shouldn't be null, because "qualifier" isn't required parameter  